server/src/main/java/org/elasticsearch/health/node/HealthIndicatorDisplayStringGetter.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.health.node;
+
+import org.elasticsearch.cluster.metadata.IndexMetadata;
+import org.elasticsearch.cluster.metadata.Metadata;
+import org.elasticsearch.cluster.node.DiscoveryNode;
+
+import java.util.Comparator;
+import java.util.Locale;
+import java.util.Set;
+
+import static java.util.stream.Collectors.joining;
+
+public class HealthIndicatorDisplayStringGetter {
```

Comment:
I think this class and its public functions would need documentation

Reply:
I think the name is also a bit inaccurate - one of the functions returns a Comparator (I love we don't call it `*Util` I must say :) 
Maybe we should not mention `String` in the name? 
ie. `HealthIndicatorDisplayValues`

Reply:
Good point, I tried hard to not call it utils or helper.... Thanks for helping come up with a better name.

Reply:
Done